openquakeplatform/openquakeplatform/static/js/hrde_input_hazard_chart.js

```diff
@@ -289,70 +184,40 @@ function hazardInputD3Chart(mfdsJsonObj) {
         .attr("x", 0)
         .attr("y", -30)
         .attr("dy", ".35em")
-        .style("font-weight", "bold")
         .attr("font-size","14px")
-        .text(mappedValue2);
-
-    textTop = svg.append("text")
-        .attr("x", 0)
-        .attr("y", -15)
-        .attr("dy", ".35em")
-        .style("font-size","11px")
-        .text('');
+        .text(mappedValue2 + ' lat/lng ' + Math.round(latlng.lat * 100) / 100 + '/' + Math.round(latlng.lng * 100) / 100 );
 
-    $('#chartDialog').append('<div>* Magnitude Frequency Distributions</div>');
-    $('#chartDialog').append('<div id="downloadCurve"><font color="blue">Download Curve</font></div>');
-    $('#downloadCurve').on("hover", function(){
-        $(this).css("cursor", "pointer");
+    $('#chartDialog').append('<div id="downloadCurve"><p style="color: blue;">Download Curve</p></div>');
+    $('#downloadCurve').on('hover', function(){
+        $(this).css('cursor', 'pointer');
     });
 
-    var h = $("#chartDialog").height();
-    h = h + 40;
-    $("#chartDialog").css({"height": h+"px"});
-
     // Prep data for download to CSV
-    $('#downloadCurve').click(function(event) {
-        var csvHeader = selectedCurves;
-        var csvData = [];
+    $('#downloadCurve').click(function() {
+        // access the mag and occurance rate arrays
+        var magsArray, occurRatesArray;
+        for (var k in mfdsJsonObj) {
+            magsArray = mfdsJsonObj[k].mags;
+            occurRatesArray = mfdsJsonObj[k].occur_rates;
+        }
 
-        //csvData = csvData.concat(csvHeader);
-        csvData.push("mfds");
-        csvData.push("binWidth");
-        csvData.push("minMag");
-        csvData.push("occurRate");
-        csvData.push("mags");
-        csvData = JSON.stringify(csvData);
-        var lineBreak = "lineBreak";
-        csvData = csvData.concat(lineBreak);
-        var quotationMark = '"';
+        var lat = latlng.lat;
+        var lng = latlng.lng;
 
+        var csvData = [];
+        csvData = csvData.concat('magnitude');
+        csvData = csvData.concat('occurrence_rate');
+        csvData = csvData.concat('lon');
+        csvData = csvData.concat('lat');
+        csvData = JSON.stringify(csvData);
+        var lineBreak = 'lineBreak';
+        csvData += lineBreak;
+        csvData += '"' + magsArray + '","' + occurRatesArray + ',' + lng + ',' + lat;
         csvData = csvData
             .replace(/lineBreak/, '\r\n')
```

Comment:
this was a strange issue, if I used \r\n directly it would not work. With this hack it works.